packages/terra-form-select/src/shared/_Frame.module.scss

```diff
@@ -307,20 +324,37 @@
       border-color: var(--terra-form-select-focus-hover-border-color, #4cb2e9);
     }
 
+    &.is-incomplete:not(.is-invalid) > .display {
+      background-color: var(--terra-form-select-incomplete-background-color, #ffc);
+    }
+
+    &.is-incomplete.is-focused:not(.is-invalid) > .display {
+      background-color: var(--terra-form-select-incomplete-focus-background-color, #ffc);
+    }
+
+    &.is-incomplete:hover:not(.is-invalid) > .display {
+      background-color: var(--terra-form-select-incomplete-hover-background-color, #ffc);
+    }
+
+    &.is-incomplete.is-disabled:not(.is-invalid) > .display {
+      background-color: var(--terra-form-select-incomplete-disabled-background-color, #ffc);
+    }
+
     &.is-invalid > .display {
       background-color: var(--terra-form-select-invalid-background-color, none);
       background-image: var(--terra-form-select-invalid-background-image, none);
-      border-color: var(--terra-form-select-invalid-border-color, #dedfe0);
+      border-color: var(--terra-form-select-invalid-border-color, #e50000);
+      box-shadow: 0 0 0 1px #e50000;
```

Comment:
should this have a variable, instead of being hardcoded?

Reply:
It should of been, added the variables here https://github.com/cerner/terra-core/pull/2670/commits/418492f37795d6dfbe0171c38fef54c1f5fd7577#diff-462afbdb742ea35bac7504edbfd87d7dR76